vaadin-notification-flow-parent/vaadin-notification-flow/src/main/java/com/vaadin/flow/component/notification/Notification.java

```diff
@@ -426,7 +430,7 @@ && getElement().getNode().getParent() == null) {
                         }
                     });
         }
-        getElement().setProperty("opened", opened);
+        getElement().setProperty(OPENED_PROPERTY, opened);
```

Comment:
Personally I don't see a need to replace `"opened"` with this constant. 
In other components we are using `getElement().getProperty("opened")`.

Reply:
Agreed, however sonar complains about duplicate string literal - https://github.com/vaadin/flow-components/runs/16713099188

Reply:
OK, let's keep it this way then.